.github/pull_request_template.md

```diff
@@ -0,0 +1,6 @@
+### Pull Request Checklist
```

Comment:
Thanks for this! I think a checklist might be useful, but that should just be for confirming you have done things which aren't clear in the PR. Could you break the rest out into sections with headings, e.g the description should just be a section here you can fill in?

Reply:
Done @oliverdunk